.github/workflows/convert.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+echo "Converting all .md tutorials to .py files..."
+cd ./docs/tutorials;
+for file in *.md; do
+    if [ "$file" != "index.md" ]; then
+        jupytext --to py $file
+    fi
+done
+cd -
```

Comment:
```suggestion
popd
```
if you go with  the other suggestion you should match the `pushd` with a `popd`